docs/source/get-started/build-and-run-examples.rst

```diff
@@ -98,30 +109,32 @@ basic usage scenarios of |short_name| with DPCPP. Go to
 
          .. code-block:: bash
 
-           # Navigate to examples directory and build examples
-           cd /examples/oneapi/dpc
-           cmake -G "Unix Makefiles⁮" -DEXAMPLES_LIST=svm_two_class_thunder # This would generate makefiles for all svm examples matching passed name
-           make               # This will compile and run generated svm examples
-           cmake -G "Unix Makefiles⁮" -DONEDAL_LINK=static # This wouldgenerate make for static version
-           make               # This will compile and run all the examples
+            # Navigate to examples directory and build examples
+            cd /examples/oneapi/dpc
+            cmake -G "Unix Makefiles⁮" -DEXAMPLES_LIST=svm_two_class_thunder # This would generate makefiles for all svm examples matching passed name
+            make               # This will compile and run generated svm examples
+            cmake -G "Unix Makefiles⁮" -DONEDAL_LINK=static # This wouldgenerate make for static version
+            make               # This will compile and run all the examples
```

Comment:
I haven't tried the windows version, but the linux one only compiles the examples without running them.